lib/membrane_rtsp/session/logic.ex

```diff
@@ -21,80 +20,30 @@ defmodule Membrane.RTSP.Session.Manager do
             realm: String.t() | nil,
             nonce: String.t() | nil
           }
+
+    @type auth_t() :: nil | :basic | {:digest, digest_opts()}
+
     @type t :: %__MODULE__{
-            transport: Transport.t(),
+            transport: any(),
```

Comment:
Why is this `any()`

Reply:
Because transport can be `any()`. It depends on the implementation of the Transport behaviour. Of course, to be a little more natural I can define a type `Transport.t()` and set it to be an alias for `any()`